docs/migrating-from-knative-build.md

```diff
@@ -1,69 +1,57 @@
-  BuildTemplates did not specify input resource requirements, and just assumed
-  whatever source was available.
-
-* Input resources must specify a `name`, which is the directory within
-  `/workspace` where the resource's source will be placed. So if you specify a
-  `git`-type resource named `foo`, the source will be cloned into
-  `/workspace/foo` -- make sure to either set `workingDir: /workspace/foo` in
-  the Task's `steps`, or at least be aware that source will not be cloned into
-  `/workspace` as was the case with Knative Builds. See [Controlling where
-  resources are
-  mounted](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#controlling-where-resources-are-mounted)
-  for more information.
-
-* TaskRuns which specify a PipelineResource to satisfy a Task's `input.resources`
-  can do so either by referencing an existing PipelineResource resource in its
-  `resourceRef`, or by fully specifying the resource in its
-  [`resourceSpec`](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md#providing-resources).
-
-* Because of how steps are serialized without relying on init containers, steps
-  should specify a `command` instead of
-  [`entrypoint`](https://github.com/tektoncd/pipeline/blob/master/docs/container-contract.md#entrypoint)
-  and `args`. If the image's `entrypoint` isn't defined, Tekton will attempt
-  to determine the image's entrypoint. This field can be a list of strings,
-  so instead of specifying `args: ['foo', 'bar']` and assuming the image's
-  entrypoint (e.g., `/bin/entrypoint`) as before, you can specify
-  `command: ['/bin/entrypoint', 'foo, bar']`.
-
-## Example
-
-### BuildTemplate -> Task
-
-Given this example BuildTemplate which runs `go test`*:
+# Migrating from Knative Build
+
+Tekton Pipelines is the technological successor to [Knative Build](https://github.com/knative/build). Tekton
+entities are based on Knative Build's entities but provide additional flexibility and reusability. This page
+explains how to convert your Knative Build entities to Tekton entities of equivalent functionality.
+
+The table below lists the mapping between the old and new entities:
+
+| **Knative Build Entity** | **Tekton Entity**  |
+|--------------------------|--------------------|
+| `Build`                  | `TaskRun`          |
+| `BuildTemplate`          | `Task`             |
+| `ClusterBuildTemplate`   | `ClusterTask`      |
+
+## Conversion guidelines
+
+Follow the guidelines below when converting your Knative Build entities to Tekton entities:
+
+* All `steps` must specify a `name` value.
+
+* `BuildTemplate` [parameters](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#parameters)
+  now reside inside the `input.params` field of a `Task`. In a related change, parameter placeholders, such as `${FOO}`,
+  now follow the format `$(input.parameters.FOO)`. For more information, see [Using variable substitution](tasks.md#using-variable-substitution).
```

Comment:
s/$(input.parameters.FOO)/$(params.FOO)/

Reply:
Not updating this since we're deleting the doc.